frontend2/src/views/Resources.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { BC23_RESOURCES } from "../content/bc23";
+import DocumentationPage from "../components/DocumentationPage";
+
+const Resources = (): JSX.Element => {
```

Comment:
Ditto the comment above about this page eventually rendering the appropriate year's resources based on the URL's `episodeId`? If that was already the plan then disregard.

Reply:
yeah, we'll render the proper year later. that change will likely come in the same pr as the change that adds the actual 2023 / 2024 episode specific text